event-processing/event-processing-template.yml

```diff
@@ -123,6 +124,13 @@ Resources:
             Action:
               - 'kms:Decrypt'
             Resource: '*'
+          - Effect: "Allow"
+            Principal:
+              AWS: "{{resolve:ssm:TxMASandboxAuditAccountARN}}"
```

Comment:
Looking at https://github.com/alphagov/di-txma-config/blob/main/audit/config-template.yml and I don't see the sandbox environment there, so how is sandbox being deployed?

Reply:
They have created their own template for their work, and will integrate at a later stage. Was this not what was planned?